apps/api/src/mail/mail.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { FormEntity } from 'src/form/entity/form.entity';
+import { MailService } from './mail.service';
+
+@Module({
+  imports: [TypeOrmModule.forFeature([FormEntity])],
```

Comment:
Nest noob here, does this mean MailModule uses Typeorm? I don't see it used in the service file

Reply:
Good catch. We are not using the entity in the mail. I'll remove it.